zone/task_client_state.cpp

```diff
@@ -1599,6 +1599,23 @@ int ClientTaskState::IsTaskCompleted(int task_id)
 	return 0;
 }
 
+int ClientTaskState::AreTasksCompleted(const std::vector<int>& task_ids)
+{
+	if (!(RuleB(TaskSystem, RecordCompletedTasks))) {
+		return -1;
+	}
+
+	int tasks_completed = 1;
```

Comment:
Strange we didn’t just make this a bool along with the return type

Reply:
Yeah, from what I see they used `-1` for disabled, `0` for fail, `1` for success.

Reply:
I can change this and the other method to a bool if we’d prefer. I doubt anyone even knows the `-1` is a thing or has ever used it in a script because why would they check for that value with the task methods if they have task system disabled?

Reply:
Changed.